charts/mlrun-ce/templates/NOTES.txt

```diff
@@ -1,8 +1,13 @@
+{{- if .Values.global.admin_install.enabled }}
+Admin Installation Completed!
+{{- end}}
+{{- if eq .Values.global.admin_install.enabled false}}
 You're up and running !
```

Comment:
Is this on purpose? You don't want to show the endpoints when doing admin install? why?

Reply:
Change to print only if one service is enable